.github/workflows/release.yml

```diff
@@ -19,7 +19,17 @@ jobs:
         run: yarn bootstrap
       - name: Run tests.
         run: yarn test:patchset
-      - name: Release to NPM
+      - name: Release snapshot to NPM
+        # lerna ususally fails in releasing, but releasing snapshots is too complicated
+        # without it, so we use lerna for snapshot and native npm-cli commands for releases
```

Comment:
I think the point of this comment would be to explain the "is too complicated" part, so we remember in few months/years what the exact issue was here, saving a few hours of (re)investigating. Could you please write a more detailed explanation?

Reply:
Added more explanation to publish.js